kythe/go/indexer/indexer.go

```diff
@@ -48,17 +49,19 @@ import (
 	"path/filepath"
 	"strconv"
 	"strings"
-
```

Comment:
Remove the leading whitespace from this line to make the linter happy

Reply:
Done - is there a way to run the linter before committing so the checks don't unnecessarily fail?